solidity/dashboard/src/css/tooltip.less

```diff
@@ -115,20 +115,9 @@
         position: relative;
         
         .tooltip {
-            &__trigger {
```

Comment:
Look into the styles of the tooltip. For big numbers it doesn't show full number and it does not look appealing for the eye because it's missing padding on the right 😛 

Example:
![image](https://user-images.githubusercontent.com/40306834/115867083-e6131200-a43a-11eb-83f0-6868e0675210.png)

Real value of totalKEEP:
![image](https://user-images.githubusercontent.com/40306834/115867100-ed3a2000-a43a-11eb-9c44-c5b6bc202058.png)